.travis.yml

```diff
@@ -3,17 +3,20 @@ sudo: false
 install:
   - mkdir bin
 
-matrix:
-    include:
-        - d: dmd-2.079.1
-        - d: dmd-2.078.3
-        - d: dmd-2.077.1
-        - d: ldc-1.8.0
-        - d: ldc-1.7.0
-        - d: ldc-1.6.0
+d:
+  - dmd
+  - dmd-2.080.0
+  - dmd-2.079.1
+  - dmd-2.078.3
+  - dmd-2.077.1
+  - ldc
+  - ldc-1.8.0
+  - ldc-1.7.0
+  - ldc-1.6.0
 
 script:
-  - dub test --build=unittest-cov --compiler=${DC}
+  - dub build -c testLibrary --compiler=${DC}
+  - dub test --build=unittest-cov
```

Comment:
`--compiler=${DC}`

Reply:
no need for it. Have dropped it from both now